### DISCUSSION TITLE
Add 3mxb format support

### DISCUSSION THREAD

**Discussion by otofoto** [cnr-isti-vclab/meshlab#1148] [Ideas]
Would be useful to have support also for 3mxb open format mesh editing capabilities. 
3mxb format is extension to OpenCTM so could be rather easy to implement with existing ctm support.

[3mxb specification](https://docs.bentley.com/LiveContent/web/ContextCapture%20Help-v18/en/GUID-2A3C8541-EF29-4F17-8999-D3398C8E7192.html)

[Example 3mxb](https://d3h9zulrmcj1j6.cloudfront.net/Graz_3MX_V4/Scene/Data/Tile_p003_p001/Tile_p003_p001_L20_000310.3mxb)
[](https://d3h9zulrmcj1j6.cloudfront.net/Graz_3MX_V4/Scene/Data/Tile_p003_p001/Tile_p003_p001_L20_000300.3mxb)

3mx multi-resolution mesh example:
[https://d3h9zulrmcj1j6.cloudfront.net/Graz_3MX_V4/App/index.html](url)

**Comment by alemuntoni** [comment]
Added the issue #1149 and labeled as a feature request. Don't know when it will be available
